.github/workflows/nightly_build_and_test.yml

```diff
@@ -0,0 +1,83 @@
+name: Build and test
+
+on:
+  schedule:
+    - cron: "0 19 * * *" # run at 4 AM JST
+  workflow_dispatch:
+
+jobs:
+  build-and-test:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        ros_distribution:
+          - foxy
+          - galactic
+          - rolling
+
+        include:
+          # Foxy Fitzroy (June 2020 - May 2023)
+          - docker_image: ubuntu:focal
+            ros_distribution: foxy
+
+          # Galactic Geochelone (May 2021 - November 2022)
+          - docker_image: ubuntu:focal
+            ros_distribution: galactic
+
+          # Rolling Ridley (June 2020 -)
+          - docker_image: ubuntu:focal
+            ros_distribution: rolling
+
+    container:
+      image: ${{ matrix.docker_image }}
+
+    steps:
+      - name: Check out repo
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 2 # https://github.com/codecov/codecov-action/issues/190#issuecomment-790729633
+
+      - name: Setup ROS environment
+        uses: ros-tooling/setup-ros@v0.2
+        with:
+          required-ros-distributions: ${{ matrix.ros_distribution }}
+
+      - name: Concat build_depends.repos
+        run: |
+          curl -sSL -H "Authorization: token ${{ secrets.REPO_TOKEN }}" https://raw.githubusercontent.com/tier4/autoware.iv/main/build_depends.repos | sed '1d' >> build_depends.repos
+
+      - name: Run action-ros-ci
+        id: action_ros_ci_step
+        uses: ros-tooling/action-ros-ci@master
```

Comment:
v2?

Reply:
I use master branch for private repository.
I opened new issue for tracking. #35

Reply:
The new version was released today. It may include the changes for private repositories.
https://github.com/ros-tooling/action-ros-ci/releases/tag/0.2.1

Reply:
Yes, it's included.
![image](https://user-images.githubusercontent.com/31987104/119431280-42749480-bd4d-11eb-8674-9c938c2d4ef2.png)

Reply:
Thank you. Then, I will replace it.